src/components/RoleSelectOnboardOne/RoleSelectOnboardOne.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import {
+  TopContainerStyles,
+  MainContainer,
+  RoleOptionContainer,
+  Container,
+  AccountContainer,
+  HeaderContainer,
+  ImageContainer,
+  JoinUsHeader,
+  FooterContainer,
+} from './RoleSelectOnboardOneStyles'
+
+import { DescriptionOffer } from '../DescriptionOffer/descriptionOffer'
+import { PageFooter, Props } from '../PageFooter/PageFooter'
```

Comment:
We should be consistent with the spaces when importing stuff. Sometimes, you don't have spaces inside of the curly braces. Pretty simple though!

Reply:
Good catch.